win32ss/user/winsrv/consrv/condrv/text.c

```diff
@@ -974,6 +1074,8 @@ ConDrvWriteConsoleOutputString(IN PCONSOLE Console,
     return Status;
 }
 
+int mk_wcwidth_cjk(wchar_t ucs);
```

Comment:
Ok maybe you can place that in some header so that it doesn't get copied over and over.

Reply:
Moved. OK.